### PR TITLE
feat: Add AI customization docs and blog post

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

**Comment by ZuzanaOdstrcilova** [comment]
Languages: Add print screen "dialog - search lang", "dialog with searched lang and pencil" and change texts according to print screens.

File: platform/projects_and_organizations/ai_translation_customization.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Providing project description
+
+Project descriptions are beneficial when you have a project in a specific domain.
+You can also specify your overall tone of voice.
+For instance, if you're translating an app for golf players, you can provide a description like the following:
+
+```text title="Example"
+This is an app for golf players.
+Please use golf terminology.
+We are trying to be friendly and funny.
```
Comment (ZuzanaOdstrcilova) [inline platform/projects_and_organizations/ai_translation_customization.mdx]: We are friendly and funny.

File: platform/projects_and_organizations/ai_translation_customization.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+We are motivating the players to get better.
```
Comment (ZuzanaOdstrcilova) [inline platform/projects_and_organizations/ai_translation_customization.mdx:30]: We motivate the players to get better.

File: platform/projects_and_organizations/ai_translation_customization.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+```
+
+To provide the project description,
+- Select `Languages` in the project menu.
+- Select `AI Customization`.
+- Click the `+ Description` button in the `Project description` section.
+
+<ScreenshotWrapper
```
Comment (ZuzanaOdstrcilova) [inline platform/projects_and_organizations/ai_translation_customization.mdx:38]: Screenshot - description (not language-notes)

**Review by stepan662 — APPROVED** [review]

**Review by ZuzanaOdstrcilova — APPROVED** [review]

**Pull request merged**